File: src/main/java/com/tom/Hello.java
package com.tom;

public class Hello {
    public static void main(String[] args) {
//        System.out.println("Hello World");
//        new  Person().hello();
        //String s =new String("abcd");
        String s = "abcde";
        System.out.println(s.charAt(3));
        System.out.println(s.length());
        System.out.println(s.substring(3));
        System.out.println(s.substring(1,4));

        Person p = new Person("Tom",66.5f,1.7f);
        p.hello();
        /*p.weight = 66.5f;
        p.height = 1.7f;*/
        System.out.println(p.bmi());

        int score = 88;
        System.out.println(score <=60);
        System.out.println(score >80 && score<90);
        System.out.println(score <80 || score>90);

        char c = 'A';
        System.out.println(c>60);
        System.out.println(c>'a');
        /*int age = 19;
        Integer age2 = 20;
        char c ='c';
        Character c2 = 'a';
        byte b =120;
        float weight = 66.5f;
        double height =1.7;
        boolean adult = true;
        boolean enroll = false;
        String name ="Tom";*/
    }
}

File: src/main/kotlin/com/tom/Hello.kt
package com.tom

fun main() {
//    println("Hello Kotlin")
//    Human().hello()
    var s : String? = "abcde"
    s = null
    //println(s!!.length)
    println(s?.length)
    println(s?.get(3))
    println(s?.substring(3))

    val h = Human(66.5f,1.7f)
    h.hello()
    println(h.bmi())

    val score = 88
    println(score > 60)

    val c : Char = 'A'
    println(c.toInt() > 60)
    /*var age : Int = 19
    age = 20
    var weight = 66.5f
    var name : String
    name ="Hank"*/


}
class Human(var weight: Float,var height : Float,var name:String =""){
    init {
        println("test $weight")
    }
//    constructor(name: String,weight: Float,height: Float) : this(weight,height)
    fun bmi(): Float{
        val bmi = weight/(height*height)
        return  bmi
    }
    fun  hello(){
        println("Hello Kotlin")
    }
}